test/reference/objects/Buildings/Controls/OBC/CDL/Interfaces/BooleanOutput.json

```diff
@@ -0,0 +1,233 @@
+                                  ],
+                                  "textString": "\"%name\"",
+                                  "textColor": {
+                                    "r": 255,
+                                    "g": 0,
+                                    "b": 255
+                                  }
+                                }
+                              }
+                            ]
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            },
+            {
+              "element_modification_or_replaceable": {
+                "element_modification": {
+                  "name": "Documentation",
+                  "modification": {
+                    "class_modification": [
+                      {
+                        "element_modification_or_replaceable": {
+                          "element_modification": {
+                            "name": "info",
+                            "modification": {
+                              "equal": true,
+                              "expression": {
+                                "simple_expression": "\"<html>\n<p>\nConnector with one output signal of type Boolean.\n</p>\n</html>\""
+                              }
+                            }
+                          }
+                        }
+                      },
+                      {
+                        "element_modification_or_replaceable": {
+                          "element_modification": {
+                            "name": "revisions",
+                            "modification": {
+                              "equal": true,
+                              "expression": {
+                                "simple_expression": "\"<html>\n<ul>\n<li>\nJuly 19, 2019, by Jianjun Hu:<br/>\nOn both icon and diagram layer, added the initialScale.<br/>\nThis is for\n<a href=\\\"https://github.com/lbl-srg/modelica-buildings/issues/1375\\\">issue 1375</a>.\n</li>\n<li>\nMarch 1, 2019, by Michael Wetter:<br/>\nOn the icon layer, changed connector size and added the connector name.<br/>\nThis is for\n<a href=\\\"https://github.com/lbl-srg/modelica-buildings/issues/1375\\\">issue 1375</a>.\n</li>\n<li>\nJanuary 6, 2017, by Michael Wetter:<br/>\nFirst implementation, based on the implementation of the\nModelica Standard Library.\n</li>\n</ul>\n</html>\""
+                              }
+                            }
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          ]
+        }
+      },
+      "within": "Buildings.Controls.OBC.CDL.Interfaces",
+      "fullMoFilePath": "/Users/akprakash/Programming/modelica/modelica-buildings/Buildings/Controls/OBC/CDL/Interfaces/BooleanOutput.mo"
+    }
+  },
+  "requiredReferences": {}
```

Comment:
empty object?

Reply:
yes. some parts of the code assume it exists (even if its empty). Can discuss and change in the future if needed